Makefile

```diff
@@ -164,6 +162,7 @@ integration-in-docker:
 		-e DOCKER_CONFIG=/root/.docker \
 		-e GOOGLE_APPLICATION_CREDENTIALS=$(GOOGLE_APPLICATION_CREDENTIALS) \
 		gcr.io/$(GCP_PROJECT)/skaffold-integration
+	docker push gcr.io/$(GCP_PROJECT)/skaffold-integration
```

Comment:
why do we need this? GCB pushes the images automatically

Reply:
I tried to pull this image and it didn't exist. So I'm not sure.

Reply:
Hmm...yeah you're right - deploy/cloudbuild.yaml only pushes up to the `builder` target in that dockerfile... why do we need to push this?

Reply:
When you say `this`, do you mean the `builder` image or the `skaffold-integration` image?
In case it's the latter, we need to push it to make sure it can be used by `--cache-from` two lines up in the script

Reply:
but integration is always different for every different version of skaffold as it is only `builder` + skaffold binary + a specific CMD? can't we just use --cache-from from the builder image?